test/release/examples/primers/arrayVectorOps.chpl

```diff
@@ -1,5 +1,18 @@
 // Array Vector Operations
 
+//
+// **DEPRECATION WARNING**
+//
+// Vector operations on arrays have been deprecated for the 1.20 release, and
+// are no longer officially supported. The `list` type has been added to the
+// standard library, and is intended to provide suitable replacements for
+// these deprecated operations.
+//
+// Use the following tutorial at your own risk. 
```

Comment:
This sounds cool, but probably isn't necessary :)

Reply:
Sounds good.